data/remnant/remnant 2 cognizance.txt

```diff
@@ -193,6 +195,8 @@ mission "Remnant: Cognizance 4"
 	to complete
 		has "remnant: cognizance: found korath fleet"
 	on complete
+		event "korath rescue: aescolanus medium"
+		event "korath rescue: aescolanus medium end" 18
```

Comment:
We didn't necessarily destroy the ships here.

Reply:
Ah, I see.  Reading up on the wiki, I take it I need to add an `action` under `branch destroyed`
Or have a silent mission trigger if that branch's condition is set

Reply:
You can use the same action that is triggered if the ships are all destroyed.